modules/representation_management/app/models/representation_management/form_2122_data.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module RepresentationManagement
+  class Form2122Data < RepresentationManagement::Form2122Base
+    service_organization_attrs = %i[
+      service_organization_name
+      service_organization_representative_name
+      service_organization_job_title
+      service_organization_email
+      service_organization_appointment_date
+    ]
+
+    validates :service_organization_name, presence: true
+
+    attr_accessor service_organization_attrs
```

Comment:
This should be `attr_reader`.

Again, can we move the `attr_reader` line above the `validates` method? Just sticking to conventions.

Reply:
Same as above.